client/rest-high-level/src/main/java/org/elasticsearch/client/ClusterClient.java

```diff
@@ -41,23 +41,55 @@
     }
 
     /**
-     * Updates cluster wide specific settings using the Cluster Update Settings API
+     * Updates cluster wide specific settings using the Cluster Update Settings API.
+     * See <a href="https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-update-settings.html"> Cluster Update Settings
+     * API on elastic.co</a>
+     * @param clusterUpdateSettingsRequest the request
```

Comment:
I'm not really sure this needs a param tag but it doesn't hurt anything.

Reply:
I went for having all the param tags rather than only some of them